packages/concerto-tools/lib/codegen/codegen.js

```diff
@@ -30,9 +30,12 @@ const MarkdownVisitor = require('./fromcto/markdown/markdownvisitor');
 const ProtobufVisitor = require('./fromcto/protobuf/protobufvisitor');
 const OpenApiVisitor = require('./fromcto/openapi/openapivisitor');
 const AvroVisitor = require('./fromcto/avro/avrovisitor');
-
-
-const InferFromJsonSchema = require('./fromJsonSchema/cto/inferModel');
+const JSONSchemaToConcertoVisitor = require(
+    './fromJsonSchema/cto/jsonSchemaVisitor'
+);
+const OpenApiToConcertoVisitor = require('./fromOpenApi/cto/openApiVisitor');
+const { JsonSchemaModel } = require('./fromJsonSchema/cto/jsonSchemaClasses');
+const { OpenApiDefinition } = require('./fromOpenApi/cto/openApiClasses');
```

Comment:
What's the motivation for exposing these classes?

Reply:
Refactored ✅